airbyte-integrations/connectors/source-twilio/source_twilio/spec.json

```diff
@@ -0,0 +1,33 @@
+{
+  "documentationUrl": "https://hub.docker.com/r/airbyte/source-twilio",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Twilio Spec",
+    "type": "object",
+    "required": ["account_sid", "auth_token", "start_date"],
+    "additionalProperties": false,
+    "properties": {
```

Comment:
Why `user_agent` field is not in this file? I saw it in `integration_tests/sample_config.json` and `integration_tests/sample_config.json` files.

Reply:
It's actually no longer needed. I'll remove the references from other files.